hibernate-core/src/main/java/org/hibernate/boot/model/internal/BasicValueBinder.java

```diff
@@ -1193,6 +1193,9 @@ else if ( aggregateComponent != null ) {
 	}
 
 	public void fillSimpleValue() {
+
+		basicValue.setMember( memberDetails.toJavaMember() );
```

Comment:
I'm pretty sure you should be passing in the whole `MemberDetails` object here, not the `Member`. The idea is to avoid accessing the reflection API unless we *really need to*.